integration-tests/tests/test_cases.py

```diff
@@ -46,9 +59,18 @@ def verify_json_schema(self, response, method):
         with open(f"{option.base_dir}/schemas/{method}", "r") as schema:
             jsonschema.validate(instance=response.json(), schema=json.load(schema))
 
+
 class TransactionTestCase(RpcTestCase):
 
-    def wallet_create_multi_transaction(self):
+    @staticmethod
+    def _update_multitx_method(params, updates):
```

Comment:
it is advanced, on my opinion for better readability would be great to pass each parameter explicitly to wallet_create_multi_transaction